src/GtkUtilities.jl

```diff
@@ -44,7 +44,7 @@ include("panzoom.jl")
 using .PanZoom
 import .PanZoom: interior, fullview   # for extensions
 
-function Base.copy!{C<:Colorant}(ctx::CairoContext, img::AbstractArray{C})
+function Base.copy!{C<:Union{Colorant,Number}}(ctx::CairoContext, img::AbstractArray{C})
```

Comment:
If the packages are truly independent and don't depend on each other, then the definition of operations between them should be in a separate package that depends on both. The difficulty is in coordinating which package should have that responsibility, particularly if multiple people have conflicting ideas. It's entirely possible for them to avoid conflicting via wrapper types, just takes a bit of restraint.

Packages are fluid and constantly break one another in myriad ways, and dependency management is a big part of that. There's a bit of truth behind some of the ravings about Julia's module system not being suitable for large scale reliable software. It might not be a coincidence or historical accident that there isn't really precedent for large software ecosystems fully built on multiple dispatch. It's convenient, but like anything that relies on shared global mutable state, it's fragile.